README.md

```diff
@@ -1 +1,2 @@
-# github-workflow
\ No newline at end of file
+# github-workflow
+## To improve is to change, to chnage is to improve
```

Comment:
There's a typo after the comment, the word "change" is misspelled.

Reply:
okay, thank you!